wpilibjExamples/src/test/java/edu/wpi/first/wpilibj/examples/addressableled/RainbowTest.java

```diff
@@ -19,7 +19,6 @@ class RainbowTest {
   void rainbowPatternTest() {
     HAL.initialize(500, 0);
     try (var robot = new Robot()) {
-      robot.robotInit();
```

Comment:
Does this still make sense given that we aren't deprecating quite yet?

Reply:
>  What we should do first is update all the templates, examples, and documentation to not use/have it, and then we can deprecate it a year later.